src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/commands.py

```diff
@@ -8,26 +8,34 @@
 
 from ._client_factory import cf_container_services
 from ._client_factory import cf_managed_clusters
+from ._client_factory import cf_openshift_managed_clusters
 from ._format import aks_list_table_format
 from ._format import aks_show_table_format
+from ._format import osa_list_table_format
 from ._format import aks_upgrades_table_format
 from ._format import aks_versions_table_format
 
 
 def load_command_table(self, _):
 
     container_services_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.containerservice.operations.'
+        operations_tmpl='azure.mgmt.containerservice.v2017_07_01.operations.'
                         'container_services_operations#ContainerServicesOperations.{}',
         client_factory=cf_container_services
     )
 
     managed_clusters_sdk = CliCommandType(
-        operations_tmpl='azure.mgmt.containerservice.operations.'
+        operations_tmpl='azure.mgmt.containerservice.v2018_03_31.operations.'
```

Comment:
Has `azure.mgmt.containerservice` gone multi-API? If so, it is very likely that changes will need to be made to the core to correctly support this.

Reply:
Yes we do since we are sharing for AKS and OSA

Reply:
Can we merge this as is ? And refactor in a separated PR with the AKS team ?